src/nle.c

```diff
@@ -73,6 +73,8 @@ vt_char_color_extract(TMTCHAR *c)
     case (TMT_COLOR_WHITE):
         color = (c->a.bold) ? CLR_WHITE : CLR_GRAY; // c = 15:7
         break;
+    case (TMT_COLOR_MAX):
```

Comment:
I think this is kind of wild but iirc TMT_COLOR_MAX.... i think is actually passed in sometime (by tmt)?!?! Really wild. I may be wrong but my some distant memory is recalling a really bonkers thing with tmt around here... maybe i used to have an assert i got rid of? i cant remember

Reply:
is this a helpful comment? probably not.

Reply:
I added this b/c the compiler doesn't like not checking all enum values. If we should do something special here, let me know.